### PR TITLE
@broskoski: An admin can add authors to an Organization

### PR DESCRIPTION
**Pull request by craigspaeth**
This creates a "autocomplete list" component that should be useful for any admin list UIs in Writer (will maybe refactor the artist & artwork page featuring to use this). That in turn made it easy to build associating authors to an organization.

cap

### CODE REVIEW

File: client/components/autocomplete_list/index.coffee [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+module.exports.AutocompleteList = AutocompleteList = React.createClass
+
+  getInitialState: ->
+    loading: true, items: @props.items or []
+
+  componentDidMount: ->
+    @addAutocomplete()
+
+  addAutocomplete: ->
+    Autocomplete ?= require '../autocomplete/index.coffee'
```
Comment (broskoski) [inline client/components/autocomplete_list/index.coffee]: why require autocomplete like this?

Comment (craigspaeth) [inline client/components/autocomplete_list/index.coffee]: I was having some trouble with typeahead in tests... there's definitely a better solution to this though.

File: client/components/autocomplete_list/README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Autocomplete List
```
Comment (broskoski) [inline client/components/autocomplete_list/README.md]: :+1:

**Comment by broskoski** [comment]
Very nice, sir!

**Comment by craigspaeth** [comment]
Note to self: This PR is lacking tests, and I need to follow up on that.

**Pull request merged**